packages/sync/src/undo-manager.ts

```diff
@@ -50,10 +61,36 @@ export function createUndoManager(): SyncUndoManager {
 		/**
 		 * Add a Yjs map to the scope of the undo manager.
 		 *
-		 * @param {Y.Map< any >} ymap The Yjs map to add to the scope.
+		 * @param {Y.Map< any >} ymap                     The Yjs map to add to the scope.
+		 * @param                handlers
+		 * @param                handlers.addUndoMeta
+		 * @param                handlers.restoreUndoMeta
 		 */
-		addToScope( ymap: Y.Map< any > ): void {
+		addToScope(
+			ymap: Y.Map< any >,
+			handlers: {
+				addUndoMeta?: RecordHandlers[ 'addUndoMeta' ];
+				restoreUndoMeta?: RecordHandlers[ 'restoreUndoMeta' ];
+			}
```

Comment:
```suggestion
			handlers: Pick< RecordHandlers, 'addUndoMeta' | 'restoreUndoMeta' >
```

Reply:
Fixed both of these instances in https://github.com/WordPress/gutenberg/pull/73066/commits/1ad8dd1a943ae6328ccb70262b2916c19123768f.